### PR TITLE
Add ".exe" to windows executable path if it is not explicitly set

### PR DESCRIPTION
**Pull request by marpaia**
This is a fix for https://github.com/gobuffalo/buffalo/issues/169.

### CODE REVIEW

**Comment by markbates** [comment]
Thanks!!

File: refresh/config.go [hunk 1]
```diff
@@ -24,7 +26,13 @@ type Configuration struct {
 }
 
 func (c *Configuration) FullBuildPath() string {
-	return path.Join(c.BuildPath, c.BinaryName)
+	path := path.Join(c.BuildPath, c.BinaryName)
```
Comment (nathany) [inline refresh/config.go:29]: `path` may not be the best variable name, given the use of the path package as well (a bit ambiguous)

File: refresh/config.go [hunk 2]
```diff
@@ -24,7 +26,13 @@ type Configuration struct {
...
+	if runtime.GOOS == "windows" {
+		if !strings.HasSuffix(path, ".exe") {
```
Comment (nathany) [inline refresh/config.go:31]: this may not be a big issue, but do note that HasSuffix is case sensitive (e.g. resulting in app.EXE.exe)

**Review by nathany — COMMENTED** [review]
Thanks for the PR.

**Comment by nathany** [comment]
Whoops, looks like I'm too late to the party. That @markbates is fast! ;-)

File: refresh/config.go (hunk 1 above)
Comment (marpaia) [reply refresh/config.go:29]: ah, yeah, this is a really good call. I can submit another PR to resolve this?

File: refresh/config.go (hunk 2 above)
Comment (marpaia) [reply refresh/config.go:31]: I can submit a new PR to lower the string first.

Comment (markbates) [reply refresh/config.go:31]: If you open a pr to handle the case and change the path variable to something else would be great.

**Pull request merged**